src/index.ts

```diff
@@ -10,6 +10,7 @@ export const onPreBuild = ({
   const emailsDirectory = process.env.NETLIFY_EMAILS_DIRECTORY ?? "./emails";
 
   netlifyConfig.functions.emails = {
+    ...netlifyConfig.functions.emails,
     included_files: [`${emailsDirectory}/**`],
```

Comment:
Need to spread the included files too my friend

Reply:
unless this is something that will never need to be modified by the user?

Reply:
Yeah I don't think I would want users modifying that tbh

Reply:
hmmmm but maybe there are email assets like images